components/prism-pascaligo.js

```diff
@@ -0,0 +1,45 @@
+// Pascaligo is a layer 2 smart contract language for the tezos blockchain
+
+Prism.languages.pascaligo = {
+	'comment': /\(\*[\s\S]+?\*\)|\/\/.*/,
+	'string': [
+		{
+			pattern: /(?:'(?:''|[^'\r\n])*')+|\^[a-z]/i,
+			greedy: true
+		},
+		{
+			pattern: /(?:"(?:""|[^"\r\n])*")+|\^[a-z]/i,
```

Comment:
Couldn't we combine this pattern with the above?
This would get rid of the duplicate `\^[a-z]`.